doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contribute to the terraform plugin.
+
+So you want to be a terraform-libvirt-plugin hacker.
+
+# Where to start?
+
+Look at open issues, especially with label:
+
+- [junior job](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22junior+job%22), 
+- [help wanted](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+
+# Contribute
+
+1. [Build from source the plugin](https://github.com/dmacvicar/terraform-provider-libvirt#building-from-source)
+2. If you have issue look first on [troubleshooting](https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/doc/TROUBLESHOOTING.md)
+3. Do your code (reference issue on your pr if you fix them)
```

Comment:
Here it would be useful to reference the GitHub help section: https://help.github.com/articles/closing-issues-using-keywords/